.github/workflows/on-pull-request.yml

```diff
@@ -75,6 +75,14 @@ jobs:
           fetch-depth: "0"
           submodules: "recursive"
 
+      - name: free disk space
+        run: |
+          sudo swapoff -a
+          sudo rm -f /swapfile
+          sudo apt clean
+          docker rmi $(docker image ls -aq)
+          df -h
```

Comment:
@howydev we should add this to the `setup` action below so that it runs on publishing packages as well or any other Action we might create later